src/index.js

```diff
@@ -1 +1,36 @@
 'use strict';
+
+const fs = require('fs');
+const http = require('http');
+const zlib = require('zlib');
+const path = require('path');
+const pipeline = require('piplene');
+
+const PORT = 3000;
+
+const server = new http.Server();
+
+server.on('request', (req, res) => {
+  const url = new URL(req.url, `http://${req.headers.host}`);
+  const fileName = url.pathname.slice(1) || 'indexe.html';
```

Comment:
The file 'indexe.html' does not exist. It seems to be a typo. The correct file should be 'index.html'.